.devcontainer/quarto-computing-dependencies-ord/install.sh

```
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -e
+
+export DEBIAN_FRONTEND=noninteractive
+
+USERNAME=${USERNAME:-${_REMOTE_USER:-"automatic"}}
+
+R_DEPS=${RDEPS:-"rmarkdown"}
+PYTHON_DEPS=${PYTHONDEPS:-"jupyter,papermill"}
+JULIA_DEPS=${JULIADEPS:-"IJulia"}
```

Shellcheck (0.10.0)

[warning] JULIA_DEPS appears unused. Verify use (or export if used externally).

(SC2034)